internal/store/certificatesigningrequest.go

```diff
@@ -79,6 +80,9 @@ var (
 					Metrics: addCSRConditionMetrics(csr.Status),
 				}
 			}),
+			AggregateBy: map[string]aggregation.Aggregation{
+				"condition": aggregation.ByLabels("condition"),
+			},
```

Comment:
I like this construct -- it is elegant to just  include an AggregateBy section in the FamilyGenerator